### PR TITLE
Feature/jen labels slds

### PR DESCRIPTION
**Pull request by jjbennett**
# Critical Changes

# Changes
The Volunteers Batch Job Progress, Job Calendar and Send Bulk Email pages have now been updated to use Lightning styling to provide consistency with Lightning Experience.

# Issues Closed

### CODE REVIEW

**Comment by jjbennett** [comment]
@davidhabib - @njjc is the QA on all but the Calendar page. This PR is ready for review. Thanks!

File: src/pages/JobCalendar.page [hunk 1]
```diff
@@ -123,35 +134,35 @@
...
                                 
-                                var strCampaignId = new String(result[i].GW_Volunteers__Volunteer_Job__r.GW_Volunteers__Campaign__c);
+                                var strCampaignId = new String(result[i].{!ns}Volunteer_Job__r.Campaign__c);
                                 var strJobId = new String(jobId);
                                 var strShiftId = new String(result[i].Id);                              
                                 var strUrl;
                                 
                                 if ({!fWeb}) {
-									strUrl = '{!$Site.CurrentSiteUrl + IF(fPersonalSite, 'GW_Volunteers__PersonalSiteJobListing', 'GW_Volunteers__VolunteersJobListingFS')}' +                                     
+                                    strUrl = '{!$Site.CurrentSiteUrl + IF(fPersonalSite, 'PersonalSiteJobListing', 'VolunteersJobListingFS')}' +
```
Comment (davidhabib) [inline src/pages/JobCalendar.page]: why don't you need to include ns on these anymore?

Comment (jjbennett) [inline src/pages/JobCalendar.page]: That may have been done in error. I will take a look, thx!

**Comment by jjbennett** [comment]
@davidhabib added back the ns to the strUrl - also made additional adjustments to the VOL_SharedCode to not hard code the namespace.

File: src/classes/VOL_SharedCode.cls [hunk 2]
```diff
@@ -687,7 +687,7 @@ global with sharing class VOL_SharedCode {
                 //IsManagedCode = (token != null);
 
     			Map<String, Schema.SObjectField> M = Schema.SObjectType.Contact.fields.getMap();
-    			Schema.SObjectField fld = M.get('gw_volunteers__volunteer_status__c');
+    			Schema.SObjectField fld = M.get(StrTokenNSPrefix('volunteer_status__c'));
```
Comment (davidhabib) [inline src/classes/VOL_SharedCode.cls]: I think you just broke this!  by dynamically adding the namespace, or leaving it out when not needed, then won't the routine end up always saying IsManagedCode = true!

**Comment by davidhabib** [comment]
Is this the intended look for the about page?  the top line really needs some whitespace if you ask me.

image

File: src/classes/VOL_SharedCode.cls (hunk 2 above)
Comment (jjbennett) [inline src/classes/VOL_SharedCode.cls]: ugg yea, fixing now!

**Comment by njjc** [comment]
I'd probably center that top line as well.

**Comment by davidhabib** [comment]
On the process recurrence schedules page, the help text says click on the Begin button, but in the page its called Run Batch.

**Comment by jjbennett** [comment]
@davidhabib thanks again, made adjustments based on your feedback. Batch page now refers to the Batch Button, fixed the IsManagedCode check and spacing on the About page.
screen shot 2016-11-21 at 1 58 09 pm

**Pull request merged**